src/talk/renderer/notifications/notifications.service.js

```diff
@@ -56,7 +56,15 @@ const refreshData = async (lastETag) => {
 		BrowserStorage.setItem('status', '' + response.status)
 		if (response.status !== 204) {
 			BrowserStorage.setItem('headers', JSON.stringify(response.headers))
-			BrowserStorage.setItem('data', JSON.stringify(response.data.ocs.data.map(remapAttributes)))
+			const data = response.data.ocs.data.map(remapAttributes)
+			BrowserStorage.setItem('data', JSON.stringify(data))
+			if (data.length) {
+				/**
+				 * Notifications older than this ID will not create a native notification.
+				 * see https://github.com/nextcloud/notifications/commit/6a543679b4de8af65cc82baa233ae17ffbbbc1af
+				 */
+				BrowserStorage.setItem('notificationThresholdId', data[0].notificationId)
```

Comment:
only set when higher than the previous value (could happen when an app is temporarily disabled)